lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -116,6 +116,99 @@ class CategoryRenderer {
   }
 
   /**
+   * @param {!ReportRenderer.AuditJSON} audit
+   * @param {number} scale
+   * @return {!Element}
+   */
+  _renderPerfHintAudit(audit, scale) {
+    const extendedInfo = /** @type {!CategoryRenderer.PerfHintExtendedInfo}
```

Comment:
discussed a lot with paul on that topic, basically by porting over so much of the way things were before and deciding against a generic solution for rendering categories/groups for now (and basically harding switch statements instead) it doesn't really fit into details cleanly since details is exclusively for expanded audit information not the audit itself, it'd be nice to retackle this when we migrate audit groups/audits themselves to details